docs/BUILD-FEDORA.md

```diff
@@ -0,0 +1,29 @@
+## Instructions for building the plugin on Fedora
+
+First, you have to install the development tools:
+```
+sudo dnf groupinstall "Development Tools"
+```
+
+Then, make sure you have the dependencies of this plugin installed:
+
+```
+sudo dnf install cmake gcc-c++ ninja-build obs-studio-devel opencv-devel qt6-qtbase-devel zsh curl-devel tesseract-devel leptonica-devel
+```
+
+Clone the repository and set up the submodules:
+```
+git clone https://github.com/obs-ai/obs-ocr.git
+cd obs-ocr
+git submodule update --init
```

Comment:
inja cannot be cloned. Please change the repository URL on .gitmodules to be an HTTPS URL.